src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskAssigneeEntity.kt

```diff
@@ -0,0 +1,19 @@
+package fi.metatavu.lipsanen.projects.milestones.tasks
+
+import jakarta.persistence.*
+import java.util.*
+
+@Entity
+@Table(name = "task_assignee")
+class TaskAssigneeEntity {
+
+    @Id
+    lateinit var id: UUID
+
+    @ManyToOne
+    @JoinColumn(name = "task_id", nullable = false)
+    lateinit var taskId: UUID
+
+    @Column(nullable = false)
+    lateinit var assigneeId: String
```

Comment:
```suggestion
    lateinit var userId: String
```
maybe UserId for clarity?

Reply:
Now it has too many mentions of assigneeId 🤣 But I can still change it if you think it is necessary